docs/testguide.md

```diff
@@ -0,0 +1,82 @@
+# Testing guide
+
+The purpose of this document is to provide start guide to set up, run and extend e2e test suite.
+
+### Prerequisites/Dependencies
+
+- Install Ginkgo and Gomega 
+
+      go get github.com/onsi/ginkgo/ginkgo
+      go get github.com/onsi/gomega/...
```

Comment:
@JaydipGabani what does the `...` signify here? Is that the actual command the user needs to run?

Reply:
@djwhatle Yes!! I was also confused, but that is how they mentioned it on the documentation.